modular_skyrat/modules/sec_haul/code/guns/bullets.dm

```diff
@@ -158,204 +90,11 @@
-	icon_state = "sl-casing"
-	caliber = CALIBER_12MM
-	projectile_type = /obj/projectile/bullet/b12mm
-
-/obj/projectile/bullet/b12mm
-	name = "12.7x30mm bullet"
-	damage = 35
-	wound_bonus = 30
-	speed = 1
-
-/obj/item/ammo_casing/b12mm/rubber
-	name = "12.7x30mm beanbag slug casing"
-	desc = "A 12.7x30mm beanbag slug casing."
-	icon = 'modular_skyrat/modules/sec_haul/icons/guns/ammo_cartridges.dmi'
-	icon_state = "sr-casing"
-	caliber = CALIBER_12MM
-	projectile_type = /obj/projectile/bullet/b12mm/rubber
-	harmful = FALSE
-
-/obj/projectile/bullet/b12mm/rubber
-	name = "12.7x30mm beanbag slug"
-	damage = 10
-	stamina = 35
-	ricochets_max = 6
-	ricochet_incidence_leeway = 0
-	ricochet_chance = 130
-	ricochet_decay_damage = 0.8
-	shrapnel_type = null
-	sharpness = NONE
-	embedding = null
-
-
-/obj/item/ammo_casing/b12mm/hp
-	name = "12.7x30mm JHP casing"
-	desc = "A 12.7x30mm JHP bullet casing."
-	icon = 'modular_skyrat/modules/sec_haul/icons/guns/ammo_cartridges.dmi'
-	icon_state = "sh-casing"
-	caliber = CALIBER_12MM
-	projectile_type = /obj/projectile/bullet/b12mm/hp
-	advanced_print_req = TRUE
-
-/obj/projectile/bullet/b12mm/hp
-	name = "12mm hollowpoint bullet"
-	damage = 35
-	wound_bonus = 40
-	embedding = list(embed_chance=75, fall_chance=3, jostle_chance=4, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.4, pain_mult=5, jostle_pain_mult=6, rip_time=10)
-	weak_against_armour = TRUE
-
-/*
-*	.577S
-*	FMJ
-*/
-
-/obj/item/ammo_casing/b577
-	name = ".577 Snider bullet casing"
-	desc = "A .577 Snider bullet casing."
-	caliber = CALIBER_B577
-	projectile_type = /obj/projectile/bullet/b577
-
-/obj/projectile/bullet/b577
-	name = ".577S FMJ bullet"
 	damage = 40
-	wound_bonus = 15
-	bare_wound_bonus = 30
-	dismemberment = 15
-
-//SMARTGUN
-/obj/item/ammo_casing/smartgun
-	name = "smartgun rail frame"
-	icon = 'modular_skyrat/modules/sec_haul/icons/guns/ammo_cartridges.dmi'
-	icon_state = "smartgun"
-	desc = "A smartgun rail."
-	caliber = "smartgun"
-	projectile_type = /obj/projectile/bullet/smartgun
-	can_be_printed = FALSE
-
-/obj/projectile/bullet/smartgun
-	name = "smartgun rail"
-	icon_state = "gaussphase"
-	embedding = list(embed_chance=100, fall_chance=2, jostle_chance=0, ignore_throwspeed_threshold=TRUE, pain_stam_pct=20, pain_mult=4, rip_time=40)
-	damage = 10
-	stamina = 70
-	ricochets_max = 6
-	ricochet_incidence_leeway = 0
-	ricochet_chance = 130
-	ricochet_decay_damage = 0.8
-	shrapnel_type = /obj/item/shrapnel/bullet/smartgun
-	hitsound = 'modular_skyrat/modules/sec_haul/sound/rail.ogg'
-
-/obj/projectile/bullet/smartgun/on_hit(atom/target, blocked, pierce_hit)
-	. = ..()
-	if(ishuman(target))
-		var/mob/living/carbon/human/H = target
-		H.emote("scream")
-		H.add_mood_event("tased", /datum/mood_event/tased)
-		if((H.status_flags & CANKNOCKDOWN) && !HAS_TRAIT(H, TRAIT_STUNIMMUNE))
-			addtimer(CALLBACK(H, TYPE_PROC_REF(/mob/living/carbon, do_jitter_animation), jitter), 5)
```

Comment:
I feel like the smart gun was unique enough compared to all the rest, I'll need a stronger reason than just "I'm doing a sweep" for its removal.

Reply:
See reply to the sec_haul guns file comment.